ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml

```diff
@@ -1,25 +1,25 @@
 base_images:
   os:
-    name: ubi
+    name: builder
     namespace: ocp
-    tag: "8"
+    tag: rhel-8-base-openshift-4.6
   os-minimal:
-    name: ubi-minimal
+    name: builder
     namespace: ocp
-    tag: "8"
+    tag: rhel-8-base-openshift-4.6
```

Comment:
I'm probably out of the loop, but can you explain this change?

Am I understanding correctly that with this change, there would be no distinction between `os` and `os-minimal`? Is there a minimal version of this image?

Reply:
Downstream product builds do not build on ubi8 minimal. This change ensure that your image builds on the same base as ART.

Reply:
Is there any reason we don't consolidate and remove the `os-minimal` base image and update any references to it to use `os` since `os` and `os-minimal` are now identical.

That might help reduce confusion for our future selves. If so, that definitely seems like something that can be done in a follow-up.